src/PowerShellEditorServices/Services/PowerShell/Debugging/DscBreakpointCapability.cs

```diff
@@ -92,10 +92,13 @@ public bool IsDscResourcePath(string scriptPath)
             if (!isDscInstalled.HasValue)
             {
                 PSCommand psCommand = new PSCommand()
+                    .AddScript("$prevProgressPreference = $ProgressPreference")
```

Comment:
```suggestion
                    .AddScript("${++prevProgressPreference} = $ProgressPreference")
```

Since this is will be ran in global scope, `$prevProgressPreference` will show up in tab completion and may conflict with a user variable. This one can still show up, but less likely (and won't conflict)